ras/ServiceReport.py.data/option.yaml

```diff
@@ -0,0 +1 @@
+plugin: 'kdump,htx'
```

Comment:
What is the idea behind checking 2 plugins together ? Wouldn't it be more useful if we check one at a time ?

Reply:
@narasimhan-v it does in a sequential manner as tools give us an option we can pass multiple plugins in time, so seems me should be ok as it user wish if he wants to pass one on multiple

Reply:
@PraveenPenguin Yes, just that as default option, we can provide one option at a time. Since default yaml serves as guideline / template for users.

Reply:
ok sure